### PR TITLE
add memory options for hosts

### PR DESCRIPTION
**Pull request by ashleyschuett**
probably need to thinking about the names... :/

### CODE REVIEW

File: options.js [hunk 1]
```diff
@@ -4,6 +4,16 @@ module.exports = {
         help: "Interval on which container harmonization occurs",
         metavar: "INTERVAL",
         default: 15000
-    }
+    },
+
+    "memory" : {
```
Comment (normanjoyner) [inline options.js]: feelings on `scheduler-available-memory`?

File: options.js [hunk 2]
```diff
@@ -4,6 +4,16 @@ module.exports = {
...
+        help: 'Amount of memory host is allowed to allocate',
```
Comment (normanjoyner) [inline options.js]: "Amount of memory the containership scheduler can allocate to running containers"

File: options.js [hunk 3]
```diff
@@ -4,6 +4,16 @@ module.exports = {
...
+        metavar: 'MAX_MEMORY'
+    },
+
+     "cpu" : {
+         help: 'Number of CPUs host is allowed to use',
```
Comment (normanjoyner) [inline options.js]: "Number of CPUs the containership scheduler can allocate to running containers"

File: options.js [hunk 4]
```diff
@@ -4,6 +4,16 @@ module.exports = {
...
         default: 15000
-    }
+    },
+
+    "memory" : {
+        help: 'Amount of memory host is allowed to allocate',
+        metavar: 'MAX_MEMORY'
+    },
+
+     "cpu" : {
```
Comment (normanjoyner) [inline options.js]: `scheduler-available-cpus`?

File: options.js [hunk 5]
```diff
@@ -4,6 +4,16 @@ module.exports = {
...
+         help: 'Number of CPUs host is allowed to use',
+         metavar: 'NUMBER_OF_CPUS'
```
Comment (normanjoyner) [inline options.js]: `SCHEDULER_AVAILABLE_CPUS`

File: options.js [hunk 6]
```diff
@@ -4,6 +4,16 @@ module.exports = {
         help: "Interval on which container harmonization occurs",
         metavar: "INTERVAL",
         default: 15000
-    }
+    },
+
+    "memory" : {
+        help: 'Amount of memory host is allowed to allocate',
+        metavar: 'MAX_MEMORY'
```
Comment (normanjoyner) [inline options.js]: `SCHEDULER_AVAILABLE_MEMORY`

File: options.js (hunk 5 above)
Comment (ashleyschuett) [inline options.js]: 👍

File: options.js (hunk 6 above)
Comment (ashleyschuett) [inline options.js]: 👍

File: options.js [hunk 7]
```diff
@@ -4,6 +4,16 @@ module.exports = {
         help: "Interval on which container harmonization occurs",
         metavar: "INTERVAL",
         default: 15000
-    }
+    },
+
+    "scheduler-available-memory" : {
+        help: 'Amount of memory the containership scheduler can allocate to running containers',
```
Comment (normanjoyner) [inline options.js]: add `(MiB)` at end of description

**Pull request merged**